app/components/ArticlePage.tsx

```diff
@@ -63,6 +70,10 @@ function Article(props: ArticlePageProps) {
             ) : (
               ''
             )}
+            {/* It's not clear if only weekly emails have prevSlug and nextSlug.
+             * They're the only articles where this link makes sense. Hmm */}
```

Comment:
You're right that they're currently the only articles using this. We could consider moving the slug parsing logic you added above into `serverOnly.ts` where prevSlug and nextSlug are computed to keep all of the weekly email logic in the same spot? Then we'd pass in `weeklyYear` as a param here.

I guess an advantage to that would be if any other article type ever started to use `prevSlug` and `nextSlug` the `weeklyYear` would be independent